logtail/logtail.go

```diff
@@ -84,6 +97,13 @@ func NewLogger(cfg Config, logf tslogger.Logf) *Logger {
 		}
 		cfg.Buffer = NewMemoryBuffer(pendingSize)
 	}
+	var instanceKey *string
+	if cfg.IncludeInstanceKey {
+		keyBytes := make([]byte, 4)
```

Comment:
I'd do a bit bigger. this isn't much random.

Reply:
This should either be large enough that random collisions never happens (in which cases this would need to be size of a MD5 or SHA256 hash), or short enough where we know it can occasionally happen with sufficiently low probability and anything processing doesn't choke on duplicates.

IIUC, our use-case is the latter, where some collisions is okay. I'm a vote for a 32-bit value.

Reply:
Sure. I guess you'd need 50 million nodes cloned off a log id to have a 1% risk of a collision.

32's fine.

I'm just used to disco keys (which made me think of this whole thing) which are 32 bytes, for different reasons.

Reply:
Stayed with 32 bit, and stored it as a uint32 to avoid the string overhead.